src/content/docs/en/guides/cms/craft-cms.mdx

```diff
@@ -0,0 +1,19 @@
+---
+title: Craft CMS
+description: Add content to your Astro project using Craft CMS as a CMS
+type: cms
+service: Craft CMS
+i18nReady: true
+stub: true
+---
+
+[Craft CMS](https://craftcms.com/) is a flexible open source CMS with an accessible authoring experience. It includes its own front end, but can also be used as a headless CMS to provide content to your Astro project.
+
+## Official Resources
+
+- Check out the official Craft CMS [GraphQL API guide](https://craftcms.com/docs/5.x/development/graphql.html)
+- The official documentation for Craft's [`headlessMode` config setting](https://github.com/Thinkmill/keystatic/tree/main/templates/astro)
```

Comment:
This link doesn't go to official documentation. Did you mean for this link to go to a template example? The documentation I could find seems to be here: 

```suggestion
- The official documentation for Craft's [`headlessMode` config setting](https://craftcms.com/docs/5.x/reference/config/general.html#headlessmode)
```

Reply:
Good catch, no idea how that got into my clipboard. I don't remember even hearing of keystatic 🤔🤔

Updated